subsys/jaxrs/src/main/java/org/commonjava/indy/bind/jaxrs/util/CdiInjectorFactoryImpl.java

```diff
@@ -35,8 +41,16 @@ protected BeanManager lookupBeanManager()
         final BeanManager bmgr = CDI.current()
                   .getBeanManager();
 
-        LoggerFactory.getLogger( getClass() )
-                     .info( "\n\n\n\nRESTEasy Using BeanManager: {}\n\n\n\n", bmgr );
+        Logger logger = LoggerFactory.getLogger( getClass() );
+
+        logger.info( "\n\n\n\nRESTEasy Using BeanManager: {}\n\n\n\n", bmgr );
+
+        Set<Bean<?>> mappers = bmgr.getBeans( ObjectMapper.class );
```

Comment:
If this is diagnostic, I wonder if we should wrap the whole thing in a `if(logger.isDebugEnabled())` condition, to prevent expensive reflection / lookup if we aren't going to use the result?

Reply:
Not sure how came this code. It seems that is nothing more than debugging. Will wrap it.